tests/utils.go

```diff
@@ -2874,7 +2874,7 @@ func AddExplicitPodNetworkInterface(vmi *v1.VirtualMachineInstance) {
 	vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
 }
 
-// Block until the specified VirtualMachineInstance reached either Failed or Running states
+// WaitForVMIStartOrFailed blocks until the specified VirtualMachineInstance reached either Failed or Running states
```

Comment:
Personally, I don't like this standard (it's duplicating the function name!) but that's the standard, so it wfm.